doc/input-output-reference/src/overview/group-zone-equipment.tex

```diff
@@ -212,14 +212,18 @@ \subsubsection{Inputs}\label{inputs-1-052}
 \begin{lstlisting}
```

Comment:
The figure above should use actual object names so that users better understand the connections.